Jasmine/LevelDesigner/ViewModels/LevelDesignerViewModel.swift

```diff
@@ -33,6 +33,20 @@ class LevelDesignerViewModel {
         return selectedPhrases[gameType]?.count ?? 0
     }
 
+    /// Adds all the phrases in levels associated with `levelInfos` to the selected phrases. 
+    ///
+    /// - Parameter levelInfos: Levels to import the phrases from
+    /// - Precondition: Levels must be of the same game type
+    func addToSelectedPhrases(from levelInfos: [GameInfo]) {
+        guard let gameType = levelInfos.first?.gameType else {
+            return
+        }
+        assert(levelInfos.map { $0.gameType }.isAllSame, "Selected levels do not have the same game type")
```

Comment:
that means the levels in the levelimportvc should be filtered according to types